packages/components/src/components/transfer-amount-input/transfer-amount-input.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import Input from '../input';
+
+type TTransferAmountInput = {
+    currency: string;
+    decimalPoints?: number;
+    disabled?: boolean;
+    initialValue?: number;
+    label?: React.ReactNode;
+    maxDigits: number;
+};
+
+const TransferAmountInput = ({
```

Comment:
```suggestion
const AmountInput = ({
```

or `CurrencyInput`. It shouldn't be specific to Transfer.